styles/start.css

```diff
@@ -0,0 +1,42 @@
+@import url('https://fonts.googleapis.com/css2?family=Archivo+Narrow:wght@400;700&display=swap');
```

Comment:
Can't find the start.css to be linked anywere, is it still in use?

Reply:
File was deleted but changes weren't pushed.